test/dotnet-new.Tests/GivenThatIWantANewApp.cs

```diff
@@ -14,58 +14,65 @@ namespace Microsoft.DotNet.New.Tests
     public class GivenThatIWantANewApp : TestBase
     {
         [Fact]
-        public void When_dotnet_new_is_invoked_mupliple_times_it_should_fail()
+        public void WhenDotnetNewIsInvokedMultipleTimesItShouldFail()
         {
-            var rootPath = TestAssetsManager.CreateTestDirectory().Path;
+            var testDirectory = TestAssets.CreateTestDirectory();
 
-            new TestCommand("dotnet") { WorkingDirectory = rootPath }
+            new TestCommand("dotnet")
+                .WithWorkingDirectory(testDirectory)
                 .Execute("new");
 
-            DateTime expectedState = Directory.GetLastWriteTime(rootPath);
+            var expectedState = testDirectory.LastWriteTime;
 
-            var result = new TestCommand("dotnet") { WorkingDirectory = rootPath }
+            var result = new TestCommand("dotnet")
+                .WithWorkingDirectory(testDirectory)
                 .ExecuteWithCapturedOutput("new");
 
-            DateTime actualState = Directory.GetLastWriteTime(rootPath);
+            var actualState = testDirectory.LastWriteTime;
 
             Assert.Equal(expectedState, actualState);
 
-            result.Should().Fail()
-                  .And.HaveStdErr();
+            result
+                .Should().Fail()
+                     .And.HaveStdErr();
         }
  
         [Fact] 
         public void RestoreDoesNotUseAnyCliProducedPackagesOnItsTemplates() 
         { 
             var cSharpTemplates = new [] { "Console", "Lib", "Web", "Mstest", "XUnittest" }; 
  
-            var rootPath = TestAssetsManager.CreateTestDirectory().Path; 
-            var packagesDirectory = Path.Combine(rootPath, "packages"); 
+            var testDirectory = TestAssets.CreateTestDirectory();
```

Comment:
looking at this - perhpaps TestAssets should inherit from DirectoryInfo, that could simplify a lot of things I believe